### ISSUE TITLE
(error) FAILED to execute command: composer create-project laravel/laravel:^11.0 ProjectYX

### ISSUE THREAD

**Issue by t0nka** [VemtoOrg/vemto2-issues#132]
**Error Message:**
(error) FAILED to execute command: composer create-project laravel/laravel:^11.0 ProjectYX

**Stack:**
```
Error: spawn /bin/sh ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
```

---
UI Stack: Breeze
Database MySQL
Vemto 2.0.10
darwin 23.4.0
Locale: de
App version: 2.0.10

**Comment by TiagoSilvaPereira** [comment]
Hi @t0nka please upgrade to Vemto 2.0.12 to see the correct error message

**Comment by t0nka** [comment]
i found the problem,

my project folder was not generated by vemto.

the screen of macos didnt allowed me to create a new folder.

so i created an empty folder by myself.

and vemto created a new project

**Comment by t0nka** [comment]
**Error Message:**
(error) FAILED to execute command: composer create-project laravel/laravel:^11.0 Foo

**Stack:**
```
Error: spawn /bin/sh ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
```

---

Vemto 2.0.12
darwin 23.4.0
Locale: de
App version: 2.0.12

**Comment by t0nka** [comment]
it s the missing check, if the new project path exists. if not existing, vemto / composer crashes

**Comment by TiagoSilvaPereira** [comment]
Hi @t0nka sorry, I did not understand the issue. Could you please send some steps so that I could reproduce and fix it?

**Comment by t0nka** [comment]
<img width="791" alt="image" src="https://github.com/TiagoSilvaPereira/vemto2-issues/assets/4823546/741d8a71-0a67-4a62-b771-807ddb0b9ef1">

1) add the project path "/Users/stefan/dev/foo"
2) add app name
3) click "create app"

if the path "/Users/stefan/dev/foo" _dont_ exist, vemto will crash
else if the path "/Users/stefan/dev/foo" exist, vemto will _not_ crash

**Comment by TiagoSilvaPereira** [comment]
@t0nka perfect, thanks for reporting

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.14
